File: Practica2/Java/Main.java
import javax.swing.*;
public class Main {

    public static void main(String[] args) {
        
        Punto p1 = new Punto(50, 50);
        Punto p2 = new Punto(200, 200);
        Punto centro = new Punto(150, 100);

        
        Linea linea = new Linea(p1, p2);
        Circulo circulo = new Circulo(centro, 50);

        
        System.out.println(linea);
        System.out.println(circulo);

        
        JFrame frame = new JFrame("Dibujo con Java");
        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
        frame.setSize(400, 400);
        frame.setContentPane(new Dibujo(linea, circulo));
        frame.setVisible(true);

File: Practica2/python/main.py
from punto import Punto
from linea import Linea
from circulo import Circulo
import turtle

p1 = Punto(0, 0)
p2 = Punto(100, 100)

print(p1.coord_cartesianas())
print(p1.coord_polares())

t = turtle.Turtle()
screen = turtle.Screen()

linea = Linea(p1, p2)
print(linea)
linea.dibujaLinea(t)

circulo = Circulo(p1, 141)
print(circulo)
circulo.dibujaCirculo(t)

screen.exitonclick()